app/controllers/report_controller.rb

```diff
@@ -2,16 +2,16 @@ class ReportController < ApplicationController
   protect_from_forgery with: :null_session
 
   def index
-    case params[:sort_column]
-    when 'per_core'
-      @reports = Report.all.order(Arel.sql('current * cpus * cores_per_cpu'))
-    when 'per_ram'
-      @reports = Report.all.order(Arel.sql('current * ram_units * ram_capacity_per_unit'))
-    when 'time'
-      @reports = Report.all.order(:created_at)
-    else
-      @reports = Report.all
-    end
+    # case params[:sort_column]
+    # when 'per_core'
+    #   @reports = Report.all.order(Arel.sql('current * cpus * cores_per_cpu'))
+    # when 'per_ram'
+    #   @reports = Report.all.order(Arel.sql('current * ram_units * ram_capacity_per_unit'))
+    # when 'time'
+    #   @reports = Report.all.order(:created_at)
+    # else
+    #   @reports = Report.all
+    # end
```

Comment:
Can this code be removed? Or is this a temporary change?

Reply:
I don't want to make to many changes on the backend implementations. It could be left to Matt to refactor and tidy up the codes.

Reply:
I'll look through this PR in a moment, this code specifically can be removed. When I first made the page I added some simple and somewhat crude sort buttons, I imagine they'll be quickly made obsolete by this PR so this is probably a good time to get rid of them

Reply:
Tbh if it was me I'd rather the code wasn't left messy! In this situation, this code is no longer works, right? You're reading the data directly with an api request, so this code is deprecated?

Reply:
I'm working on a commit ATM to clean up the backend, there's a little extra stuff associated with that code. Although removing it doesn't break anything, there's still routes pointing to it and those routes are still used somewhere so it makes sense to clean it all up at once

Reply:
I was not sure if Matt would like to keep these codes as a reference. They have been removed now.